### PR TITLE
Prevent ASCII error for encode/decode in utf8 into converter

### PR DESCRIPTION
**Pull request by bsuttor**
I preferred adding safe_utf8(s) method for not adding plone.dexterity dependency.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,17 +4,10 @@ Changelog
...
 ------------------
 
-Breaking changes:
-
-- *add item here*
-
-New features:
-
-- *add item here*
-
```
Comment (gforcada) [inline CHANGES.rst]: @bsuttor please keep these headers, when doing a release with plone.releaser they are automatically removed, but while it is still not there could be any potential break or new feature added and no header would be available.

Comment (bsuttor) [inline CHANGES.rst]: Ok, no problem, I will let it next time, I didn't know that

**Comment by jensens** [comment]
Please restore the mentioned headers, otherwise no merge possible,

**Comment by laulaz** [comment]
@jensens Can we merge this one ?

**Comment by jensens** [comment]
@laulaz yes, just trigger jenkins and if all is green I'll merge.

**Comment by laulaz** [comment]
@jensens now you can :+1:

**Pull request merged**